browser/actors/FathomChild.jsm

```diff
@@ -13,7 +13,7 @@ var EXPORTED_SYMBOLS = ["FathomChild"];
 //   "resource://gre/modules/Fathom.jsm"
 // );
 
-const { Fathom } = ChromeUtils.import("resource:///modules/Fathom.jsm");
+const { Fathom } = ChromeUtils.import("resource://gre/modules/Fathom.jsm");
```

Comment:
My thinking is we should keep Fathom desktop only for now; especially for a demo.

Sure ideally some time in the future, we want it to be available on mobile, but that is opening up a whole new can of worms.
* We have several different, separate Firefox applications, Fennec and Fenix on Android and Firefox for iOS.
* These are written in different languages with bindings and varying levels of support for Firefox APIs.
  * For example, I know that many longstanding WE APIs have only been partially implemented if implemented at all in Fennec.
* Further, I know Fennec is single process while Fenix is multiprocess, and we can probably assume their Fission status is different with respect to one another and compared to desktop (i.e. our use of `JSWindowActors`).

Reply:
Does this suggest any code changes to you? The last I heard from mattn was "toolkit/modules would traditionally be the place if mobile would use it but with GeckoView that's kinda changed… I would suggest toolkit/modules/ still though", suggesting (to me) it should be where it is regardless. How about you?

Reply:
Yes I'm suggesting a code change -- specifically, I am suggesting keeping all of our files in `browser` rather than `toolkit`. For example, we could make a `browser/components/fathom` directory and keep our files in there (outside of the `FathomChild` `JSWindowActor`). I'm not sure if that's the best place for it, but I see examples like the [`newtab` component](https://searchfox.org/mozilla-central/source/browser/components/newtab) which has a `vendor` subfolder.

Reply:
This of course, is non-blocking for the demo. I just think post-demo we will want to put it in `browser`.

Reply:
I'm still having a hard time deciding where the final resting place will be, probably because we don't know yet what our optimization techniques will require (process-wise, etc., as you said). How about we leave it as is for now, and we can easily move it around later if we need to rule out mobile?

Reply:
Sure that's fine; we can revisit this later. Maybe file a follow-up issue with a label of `post-demo` or similar?